src/common/playlists/PlayList.jsx

```diff
@@ -37,7 +44,12 @@ const PlayList = () => {
             found_plays.push(res_play);
           }
         });
-        setPlays(found_plays);
+        if (sortBy === 'Random') {
+          const random_plays = found_plays.sort(() => Math.random() - 0.5);
+          setPlays(random_plays);
+        } else {
+          setPlays(found_plays);
+        }
```

Comment:
Suggestion to use camel casing for variables in JS files. `foundPlay` and `randomPlays`.